db/seeds.rb

```diff
@@ -6,14 +6,74 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+VALID_EMOJIS = ["heart_eyes", "beer", "clap", "sparkling_heart", "heart_eyes_cat", "dog"]
```

Comment:
Technically we could reuse the version of this constant from the `Board` class by just referencing `Board.VALID_EMOJIS.sample` in the code later on in this file.